### PR TITLE
Add notes from May

### PR DESCRIPTION
**Pull request by encukou**
I still had the window open from adding the April notes :)

----
📚 Documentation preview 📚: https://docs-community--115.org.readthedocs.build/

### CODE REVIEW

File: docs/monthly-meeting/2024-05.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+## Discussion
+
+- [Petr] Found some older PRs that didn't get to Beta
+  - Intersphinx in CPython docs -- [python/cpython#117540](https://github.com/python/cpython/pull/117540)
+      - Redistributors build docs while offline.  This change shouldn't be merged during beta. On May 8th main will be 3.14 alpha, it could be merged then.
+      - Would also be good to map urls to local file paths for offline use.  Open edX might have something like this to keep tests from breaking as different docs update at different rates.
+  - Module page titles starting with a link to themselves -- [python/cpython#117099](https://github.com/python/cpython/pull/117099)
+      - Didn't want to ping 29 code owners.
+          - But they get pings a lot anyway.
+      - A custom sphinx plugin could prevent links in all section headers instead.
```
Comment (hugovk) [inline docs/monthly-meeting/2024-05.md:54]:
```suggestion
      - A custom Sphinx plugin could prevent links in all section headers instead.
```

File: docs/monthly-meeting/2024-05.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        - This leads to the question of whether a human or an algorithm should make this decision.
+      - Practically: merging this would be a good thing
+      - Can we have a linter for links in section heads?
+          - We could add a rule to [Sphinx Lint](https://pypi.org/project/sphinx-lint/).
+  - Merge versionchanged directives for the same version -- [python/cpython#116456](https://github.com/python/cpython/pull/116456)
+    - We agree unanimously that the status quo is better; we'll close the PR.
+    - Serhyi's other contributions on the issue are great, though!
+
+
+- [Hugo] RTD migration work continuing: [python/docs-community#5](https://github.com/python/docs-community/issues/5), have a WIP version and language selector at https://cpython-previews.readthedocs.io/en/main/
```
Comment (hugovk) [inline docs/monthly-meeting/2024-05.md:65]: For some reason plain URLs aren't linked:

```suggestion
- [Hugo] RTD migration work continuing: [python/docs-community#5](https://github.com/python/docs-community/issues/5), have a WIP version and language selector at [https://cpython-previews.readthedocs.io/en/main/](https://cpython-previews.readthedocs.io/en/main/)
```

**Review by hugovk — APPROVED** [review]
Thank you!

File: docs/monthly-meeting/2024-05.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+(Name / `@GitHubUsername` *[/ Discord, if different]*)
+
+- Ned Batchelder / `@nedbat`
+- Petr Viktorin / `@encukou`
+- Ryan Duve / `@ryan-duve`
+- Eric
+- Hugo van Kemenade / `@hugovk`
+- CAM
+- Erlend
```
Comment (ezio-melotti) [inline docs/monthly-meeting/2024-05.md:27]:
```suggestion
- Eric
- Hugo van Kemenade / `@hugovk`
- CAM / `@CAM-Gerlach`
- Erlend Aasland / `@erlend-aasland`
```

**Pull request merged**